src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -304,6 +294,18 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
                     el.innerHTML = processedValue;
 
+                    if (domUtils.isBodyElement(el)) {
```

Comment:
`onBodyContentchanged` is called asynchronously.
There is test case where we need to sync flow - https://github.com/DevExpress/testcafe-hammerhead/pull/1150/files#diff-52837961726917bc92aefa76983a3eceR694